README.md

```diff
@@ -1,2 +1,60 @@
-# ffc-axe
-AXE Accessibility testing Docker image
+# FFC AXE
+
+Docker image for running [axe-core/cli](https://www.npmjs.com/package/@axe-core/cli)
+
+## Usage
+
+This image was built to support running AXE acceptance tests against a containerised web application orchestrated by Docker Compose. Example:
+
+```
+version: '3.7'
+
+services:
+  ffc-demo-web:
+    command: npm run start:watch
+    build:
+      target: development
+    image: ffc-demo-web-development
+    environment:
+      NODE_ENV: test
+    volumes:
+      - ./test-output/:/home/node/test-output/
+
+  axe:
+    image: ffc-axe
```

Comment:
You'd probably want `defradigital/` as the image reference as that is what it will be when published.  It won't work as is.